### ISSUE TITLE
Single page URL

### ISSUE THREAD

**Issue by OlgaKrebs** [seek4science/seek#1235]
opening "Single page" view one has an URL https://testing.sysmo-db.org/single_pages/116  and this stays-  even one  navigates to I-S-A elements or assets  OR to folders  - URL stays https://testing.sysmo-db.org/single_pages/116. So there is no chance to see (copy) URLs of any asset in this project from single page view?.  

May be this should be another issue (but cause it is abut visualisation/navigation I first add it here) - If one goes to e.g. Investigation and click "edith" or "manage" Investigation and does editing, confirms - and gets to the Investigation page (single page view is gone) and to get there again one needs 3 more clicks

**Comment by floradanna** [comment]
Yes, correct. We are aware of this issue. We agree that URL should change, indeed. We plan to address this issue in the future.
